pages/service/profiles/users-and-profiles.mdx

```diff
@@ -30,4 +30,23 @@ The next step is allowing the user to obtain the challenge we created within the
 * If your game is running on a console, PC or within a browser - we recommend you to encode the challenge in a QR code
 * If your game is running on iOS or Android, you could pass the challenge down a deeplink that opens up the Beam companion app
 
-The user will scan the QR code with the companion app, which is how we create a relationship between the user in Beam and the player profile you've registered. By providing a `callbackUrl` when creating a Connection request, we'll be able to provide you with an incoming webhook telling you that a profile was linked up to a user. At the moment of writing, the callback impelmentation isn't in place yet, but we'll be providing details on the exact data model you will receive before the launch of the Beam app early Q4.
\ No newline at end of file
+The user will scan the QR code with the companion app, which is how we create a relationship between the user in Beam and the player profile you've registered. By providing a `callbackUrl` when creating a Connection request, we'll be able to provide you with an incoming webhook telling you that a profile was linked up to a user.
+
+The incoming webhook doesn't tell you to which user a profile is linked to, but it does confirm to you that a link was made between the app-user and your game's profile. When the incoming webhook arrives, there will be a `Signature` header included in it. The signature can be ignored completely, but allows you to verify whenever the incoming request we send you indeed is coming from Beam. The implementation for handling this signature are platform specific and you are required to use Libsodium. Below you will find an example on how to verify the signature within a Node.js based backend.
```

Comment:
can you also add a link on the `libsodium` to their webpage? https://doc.libsodium.org/